table/tables/tables.go

```diff
@@ -1346,9 +1349,11 @@ func (t *TableCommon) removeRowData(ctx sessionctx.Context, h kv.Handle) error {
 	})
 	if t.shouldAssert(ctx) {
```

Comment:
Should we handle/update the asserts for [`AddRecord`](https://github.com/pingcap/tidb/blob/a4ab759a7e41f2fcbefd2553ff93ed42d83a8f0c/table/tables/tables.go#L714)?

Reply:
`AddRecord` does not call `shouldAssert()` @bb7133